ansible/roles/postgresql/tasks/main.yml

```diff
@@ -107,6 +108,30 @@
   when:
     pg_install == True
 
+- name: Grant PostgreSQL privileges
+  sudo_user: postgres
+  postgresql_privs: type=database database="{{ item.1.db_name }}" roles="{{ item.0.username}}" privs="{{ item.1.db_priv}}"
+  with_subelements:
+    - "{{ postgres_users.values() }}"
+    - privileges
+    - flags:
+      skip_missing: True
+
+- name: Set readonly PostgreSQL roles
+  sudo_user: postgres
+  shell: "PGPASSWORD={{ postgres_users.get(item.1.readonly_master).get('password') }}
+    {{postgres_install_dir}}/bin/psql -U {{ item.1.readonly_master }} -h localhost {{item.1.db_name}} -c
+    'GRANT USAGE ON SCHEMA PUBLIC to {{ item.0.username }}; 
+    GRANT SELECT ON ALL TABLES IN SCHEMA PUBLIC to {{ item.0.username }}; 
+    ALTER DEFAULT PRIVILEGES IN SCHEMA PUBLIC GRANT SELECT ON TABLES TO {{ item.0.username }}'"
+  when:
+    item.1.readonly == True
+  with_subelements:
```

Comment:
@benrudolph I looked at this again and the issue is that `with_subelements` isn't well supported in ansible < 2.0.